lib/Cake/Model/Behavior/TranslateBehavior.php

```diff
@@ -353,11 +354,35 @@ public function beforeSave(Model $Model, $options = array()) {
 			unset($this->runtime[$Model->alias]['beforeSave']);
 		}
 		if (isset($this->runtime[$Model->alias]['beforeSave'])) {
+			$this->_cleanSaveData($Model);
 			return true;
 		}
 		$this->_setRuntimeData($Model);
+		$this->_cleanSaveData($Model);
 		return true;
 	}
+	
+/**
+ * Clean save data
+ *
+ * Prevents Model data from being saved if not in the 'base' language
+ * as defined by Config.language 
+ *
+ * @param Model $Model Model using this behavior.
+ * @return void
+ */
+	protected function _cleanSaveData(Model $Model) {
+		$locale = $this->_getLocale($Model);
+		if (empty($locale)) {
+			return true;
```

Comment:
I must've missed the notification - Shouldn't it return true to assert that the data is clean?